flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/Benchmark.java

```diff
@@ -120,7 +132,10 @@ public static void main(String[] args) throws Exception {
         } else if (commandLine.getArgs().length == 1) {
             executeBenchmarks(commandLine);
         } else {
-            printInvalidError(args);
+            System.out.println("Invalid command line arguments " + Arrays.toString(args));
```

Comment:
Would it be simpler and more consistent with the LOG.info(...) above by using the following code:

`System.out.println(String.format("Invalid command line arguments %s \n", Arrays.toString(args)))`